src/sources/aws_s3/sqs.rs

```diff
@@ -44,12 +48,12 @@ pub(super) struct Config {
     // restricted to u32 for safe conversion to i64 later
     #[serde(default = "default_poll_secs")]
     #[derivative(Default(value = "default_poll_secs()"))]
-    pub(super) poll_secs: u32,
+    pub(super) poll_secs: i32,
```

Comment:
Does it make sense to allow negative poll and visibility secs?

Reply:
not really, but this is what the SDK accepts. I just made it match. I don't have strong feeling here, if you prefer keeping is a `u32` I can change it

Reply:
Ah, yeah, I think it was originally a u32 to have it be a configuration error if they pass a negative number. Otherwise I'm not sure what the behavior would be but would guess the AWS API would return an error.